tests/test_connection.py

```diff
@@ -2322,7 +2324,8 @@ def test_dataloss_protection(node_factory, bitcoind):
 
     # l2 must NOT drop to chain.
     l2.daemon.wait_for_log("Cannot broadcast our commitment tx: they have a future one")
-    assert not l2.daemon.is_in_log('sendrawtx exit 0')
+    assert not l2.daemon.is_in_log('sendrawtx exit 0',
+                                   start=l2.daemon.logsearch_start)
```

Comment:
This is semantically the same as `wait_for_log` which starts at `logsearch_start`.